data/preset/course.yaml

```diff
@@ -0,0 +1,150 @@
+---
+defines:
+
+- &main-arrangement-slot
+  arrangement:
+    type: Arrangement.Grid
+    properties:
+      homogeneous: true
+      max-children-per-line: 3
+      min-children-per-line: 3
+    slots:
+      card:
+        shortdef: 'Card.Thumb(background-size: "center")'
+
+- &home-page
+  shortdef: Layout.Scrolling
+  slots:
+    content:
+      shortdef: 'Layout.Box(orientation: vertical)'
+      slots:
+        contents:
+        - shortdef: Layout.Overlay
+          styles:
+            - landing
+          slots:
+            content:
+              shortdef: Layout.Box
+              styles:
+                - background
+            overlays:
+            - shortdef: 'Layout.Box(orientation: vertical, halign: start)'
+              styles:
+                - intro
+              slots:
+                contents:
+                - shortdef: 'Banner.App'
+                - shortdef: 'ContentGroup.StaticText(wrap: true, wrap-mode: word-char, ellipsize: end, max-width-chars: 34, lines: 8,label: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam vehicula diam id quam ornare, eu mattis ex vulputate. Integer egestas, nisi vel tincidunt congue, mauris elit pharetra mi, sit amet accumsan urna ex eu erat. ")'
+            - shortdef: 'Banner.Dynamic(show-subtitle: true, mode: text, halign: end, valign: center)'
+        - shortdef: 'ContentGroup.ContentGroup(expand: true)'
+          slots:
+            <<: *main-arrangement-slot
+            title:
+              shortdef: 'Banner.Dynamic(halign: start, format-string: "Courses from %s")'
+            selection:
+              type: Selection.AllSets
+              slots:
+                filter: Filter.Featured
+        - shortdef: 'Layout.Box(orientation: vertical, halign: fill, hexpand: true)'
+          styles:
+            - footer
+          slots:
+            contents:
+            - shortdef: 'ContentGroup.StaticText(lines: 1, label: "More courses soon.")'
```

Comment:
This is more of a design problem than a code review problem, but what does "More courses soon" mean? Does it really belong in the preset? I've been meaning to ask this after all three of the demos you gave...